release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -44,6 +44,19 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-7-aws-c2s-region"]
+==== Installing a cluster into an AWS C2S region
+
+You can now install a cluster on Amazon Web Services (AWS) into a Commercial Cloud Services (C2S) region. Because C2S regions do not have {op-system} AMIs published by Red Hat, you must upload a custom AMI that belongs to that region. You are also required to include the CA certificates for C2S in the `additionalTrustBundle` field of the `install-config.yaml` file during cluster installation. The installation program does not support destroying a cluster deployed to a C2S region; you must manually remove the resources of the cluster.
```

Comment:
Do you think we should mention that installs to C2S must use an image registry since the cluster will not have access to the Internet? Or is it safe to assume that any using C2S will already know that the cluster will be disconnected?

Reply:
I think mentioning it would be helpful. I'll add that in :+1: